src/help/guides/extension-development/intro-to-sources.md

```diff
@@ -1,12 +1,23 @@
 ### Introduction: A Manga Reader Without Servers
-When developing a Manga Reader, first and foremost, you need to gather your content from somewhere. There are many services out there, such as WebToons or MangaDex which run their own content providing system which their applications can attach to without much issues.
 
-If you do not hold this kind of infrastructure, as Paperback does not, the app needs to be designed in a way where it can dynamically read **other** content providers.
+When developing a Manga Reader, first and foremost, you need to gather your content from somewhere. There are many
+services out there, such as WebToons or MangaDex which run their own content providing system which their applications
```

Comment:
I'm not sure to what refer "WebToons", could it be "WEBTOON"?

Reply:
WebToons are a type of long strip content made for viewing on mobile devices. WEBTOON is one company that publishes them.

Reply:
Also, I did not modify this, just formatted it, so the text is from the old GitHub wiki.

Reply:
@ConradWeiser when you wrote "WebToons", were you talking about the format or the company? The presence of MangaDex let me suppose it was the company but I could be mistaking.

Reply:
Originally it was the company/group, yeah.

Reply:
Thanks Vibrant. Then I propose to use
```suggestion
services out there, such as WEBTOON or MangaDex which run their own content providing system which their applications
```